x/dclauth/handler_test.go

```diff
@@ -131,6 +134,54 @@ func TestHandler_CreateAccount_ThreeApprovalsAreNeeded(t *testing.T) {
 	trustee3 := storeTrustee(setup)
 	_ = storeTrustee(setup)
 
+	// store account
```

Comment:
We already have `TestHandler_ReAdding_RevokedAccount`. Why do we need to do revocation in this test? The test name is `CreateAccount`, revocation here is not expected.

Reply:
Done! ✅